src/disc/Albany_AbstractDiscretization.hpp

```diff
@@ -41,39 +44,100 @@ class AbstractDiscretization
   //! Destructor
   virtual ~AbstractDiscretization() = default;
 
+  //! Get the DOF manager
+  Teuchos::RCP<const DOFManager>
+  getNewDOFManager (const std::string& fieldName) const {
```

Comment:
Are you going to rename this `getDOFManager` before merging the PR?

Reply:
Yes. I could do it now, since the old NodalDOFManager has already been removed. But I might just wait till the end for the cleanup stuff.